run.cmd

```diff
@@ -0,0 +1,27 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+if not defined VisualStudioVersion (
+  if defined VS140COMNTOOLS (
+    call "%VS140COMNTOOLS%\VsDevCmd.bat"
+    goto :Run
+  )
+  echo Error: Visual Studio 2015 required.
+  echo        Please see https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/developer-guide.md for build instructions.
```

Comment:
There's probably stuff in developer-guide.md that needs to be be changed; not necessary for this PR but we should get that done in the not-too-distant future